Google.PowerShell/Provider/GoogleCloudStorageProvider.cs

```diff
@@ -254,27 +254,32 @@ public string RelativePathToChild(string childObjectPath)
         /// <summary>
         /// Reports on the usage of the provider.
         /// </summary>
-        private IReportCmdletResults _telemetryReporter;
+        private IReportCmdletResults TelemetryReporter => _telemetryReporter.Value;
```

Comment:
The provider gets created a lot, and avoiding the disk read the reporter requires seems a good idea. I made it static, so it is only ever created once.